tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -334,4 +334,83 @@ TEST_F(MatmulATenEvaluationTest, MatmulBiasAlpha) {
 
   EXPECT_TRUE(at::allclose(out[0], out_ref));
 }
+
+TEST_F(MatmulATenEvaluationTest, Linear) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  int64_t m = 32, n = 64, k = 128;
+  std::vector<int64_t> a_shape{m, k}, b_shape{n, k}, out_shape{m, n};
+
+  auto tv0 = makeConcreteTensor(a_shape, DataType::Half);
+  auto tv1 = makeConcreteTensor(b_shape, DataType::Half);
```

Comment:
Is there any reason to make these tensors concrete?

Reply:
No. We can go with symbolic or other types too.
We need to expand the tests (Issue #2032) which can be done in this or another PR.

Reply:
@jacobhinkle I'll pin this for the next PR. Do you have any preference?

Reply:
Separate PR sounds good. Keeping the PRs small is best for velocity.